code/modules/clothing/clothing.dm

```diff
@@ -37,6 +37,8 @@
 	// copies the properties of the clothing item to the accessory, in the future, take literally almost every var from ties.dm parent object and place it in clothing parent
 	var/obj/item/clothing/accessory/new_accessory = new accessory_path(loc)
 	new_accessory.name = name
+	if(istype(src, /obj/item/clothing/yautja_cape)) // checks for Pred Cape for recoloring
+		new_accessory.color = color
```

Comment:
you may as well allow any coloured accessory to remain so, they're not typically changeable without staff intervention anyway

Reply:
Fixed